eve-api-data/src/main/java/com/eveonline/api/repo/AccountBalanceRepository.java

```diff
@@ -3,6 +3,12 @@
 
 import com.eveonline.api.data.AccountBalance;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface AccountBalanceRepository extends JpaRepository<AccountBalance, Long>
+{
+    @Query(value = "select a from AccountBalance a where a.accountKey = :accountKey and a.corporationId = :corporationId")
```

Comment:
This @Query annotation shouldn't be necessary as spring-data-jpa has a convention for these method names.  The name (if we break it up into components)

findBy (select * from apiaccountbalance)
AccountKey (where account_key = <the value of the first method parameter>)
And (and)
CorporationId (corporation_id = <the value of the second method parameter>)